Core/HLE/sceAtrac.cpp

```diff
@@ -1964,6 +1964,8 @@ static const u16 at3plusHeaderMap[][3] = {
 	{ 0x4524, 0x0, 0x0 },
 	{ 0x02E8, 0x1, 0x0 },
 	{ 0x5C24, 0x0, 0x0 },
+	{ 0x0118, 0x1, 0x0 },
+	{ 0x2224, 0x0, 0x0 },
```

Comment:
Yeah,so I think the above one is 0x0118,not 0x0180.0x2224 should make pair with 0x0118.
you can see those pair
```
{ 0x0178, 0x1, 0x0 }, { 0x2E24, 0x0, 0x0 }, 
{ 0x0178, 0x2, 0x0 }, { 0x2E28, 0x0, 0x0 },
```
```
 { 0x0230, 0x1, 0x0 },{ 0x4524, 0x0, 0x0 },
 { 0x0230, 0x2, 0x0 },{ 0x4528, 0x0, 0x0 },
```
```
{ 0x02E8, 0x1, 0x0 },{ 0x5C24, 0x0, 0x0 },
{ 0x02E8, 0x2, 0x0 },{ 0x5C28, 0x0, 0x0 },
```